src/DataStructures/DataBoxTag.hpp

```diff
@@ -27,6 +27,14 @@ struct Variables;
 }  // namespace Tags
 /// \endcond
 
+namespace Tags {
+struct DataBox {
```

Comment:
Please document that it is possible to fetch/forward a DataBox from itself by using this tag, and provide an example or description of how this is useful